style.css

```diff
@@ -23,7 +25,7 @@
 .nav-container {
   background: var(--lighter-grey);
   width: 100%;
-  display: flex;
+  display: block;
   align-items: center;
   position: fixed;
   box-shadow: 0 1.2rem 3.2rem rgba(0, 0, 0, 0.03);
```

Comment:
- [ ] It is required to stick to the Figma design as much as possible. Your overlay background is supposed to cover the whole height of the page, but this is not so on your design. Kindly ensure the overlay fills the whole page as your mobile menu items appears when the hamburger icon is clicked.

### Your Design
![your-desing](https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/26597432/e4e55b9a-4b36-4f4a-8d6e-dac47039b5d9)

### Figma Design
![figma-design](https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/26597432/ae258408-cfe4-45c5-b1f0-7b05bb1fd3cf)

- [ ] Kindly reduce the margin left for all the menu items so as to match the Figma design

### Your Design
![your-space](https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/26597432/85e60fb0-d214-4928-95cd-47435c887c09)

### Figma Design
![figma-space](https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/26597432/bcd477ea-2bea-40b6-b363-b619428b00dc)